inji-verify/src/components/Home/VerificationSection/QrScanner.tsx

```diff
@@ -67,138 +66,61 @@ function QrScanner() {
   const processFrame = useCallback(() => {
     const canvas = canvasRef.current;
     const video = videoRef.current;
-    const zoom = 1 + zoomLevel / 10;
-
-    if (canvas && video) {
-      canvas.width = canvas.clientWidth;
-      canvas.height = canvas.clientHeight;
+    if (canvas && video && video.videoWidth && video.videoHeight) {
+      canvas.width = video.videoWidth;
+      canvas.height = canvas.width * (video.videoHeight / video.videoWidth);
       const ctx = canvas.getContext("2d", { willReadFrequently: true });
-      ctx?.clearRect(0, 0, canvas.width, canvas.height);
-      const canvasCenterX = canvas.width / 2;
-      const canvasCenterY = canvas.height / 2;
-      ctx?.save();
-      ctx?.translate(canvasCenterX, canvasCenterY);
-      ctx?.scale(zoom, zoom);
-      ctx?.translate(-canvasCenterX, -canvasCenterY);
       ctx?.drawImage(video, 0, 0, canvas.width, canvas.height);
-      readBarcodeFromCanvas(canvas);
+      setTimeout(readBarcodeFromCanvas, 30, canvas);
```

Comment:
processframe is already called after a 30 ms delay do we again need this delay ?